GAE/src/locale/ui-strings.properties

```diff
@@ -1,4 +1,3 @@
-a_link_emailed = A link will be emailed to you shortly.
```

Comment:
Why is this file diferent from en.properties? keys do not follow the same convention (_ instead of \ ), the naming is diferent: why ui-strings.properties and the other en.properties, does this one not get translated?

Reply:
the ui-strings.properties is the one with the placeholders but the en.properties is the source file in transifex.  I believe its setup this way in order to not have placeholders appear for other languages but rather to have a plain english text string.